src/bot.ts

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-trailing-spaces */
+import fs from 'node:fs';
+import dotenv from 'dotenv';
+import { ActivityOptions, Client, Collection, Guild, MessageEmbed, TextChannel } from 'discord.js';
+import { Command } from './interfaces/Command';
+import { newDay, setUp } from './database';
+import { log } from './utils/log';
+import { runAtMidnight } from './utils/runner';
+dotenv.config();
+
+const client = new Client({ intents: ['GUILDS', 'GUILD_MESSAGES', 'GUILD_MESSAGE_REACTIONS'] });
+
+client.once('ready', () => {
+	log('Client is ready', 'BOT', 'green');
+
+	client.user!.setActivity({ type: 'PLAYING', name: '/adivinhar' });
+
+	const ac: ActivityOptions[] = [
+		{ name: `em ${client.guilds.cache.size} servidores`, type: 'PLAYING' },
+		{ name: '/adivinhar', type: 'PLAYING' },
+		{ name: 'suas tentativas...', type: 'WATCHING' },
+	];
+	setInterval(() => {
+		const activity = ac[Math.floor(Math.random() * ac.length)];
+		client.user!.setActivity({ name: activity.name, type: activity.type });
+	}, 900_000);
+});
+
+const cLog = client.channels.cache.get(process.env.GUILD_UPDATE_CHANNEL) as TextChannel;
+
+client.on('guildCreate', async (guild: Guild) => {
+	const { createdTimestamp, ownerId, memberCount, name, id } = guild;
+
+	const owner = await guild.fetchOwner().then(o => o.user.tag);
```

Comment:
It's kinda weird seeing `await` and `then` together, but I guess it works

Reply:
`.fetchOwner()` is a Promise, and I need to return the `user.tag`. But works yes.
Do you have any suggestion?

Reply:
Maybe 
```suggestion
	const owner = await guild.fetchOwner();
	const ownerTag = owner.tag
```

or just use owner.tag directly.
But we can leave it as-is, it's just weird in my opinion.

Reply:
I see, thanks.